src/components/Badge/Badge.jsx

```diff
@@ -0,0 +1,63 @@
+import styles from "./Badge.module.css";
+
+const Badge = ({ text, color }) => {
+  const { backgroundColor, color: textColor } = BADGE_COLORS[color];
+
+  return (
+    <span
+      className={styles.badge}
+      style={{ backgroundColor, color: textColor }}>
```

Comment:
인라인 스타일로 조건부 스타일을 주는 것 보다 클래스로 제어하는게 더 좋을 것 같아요!
스타일코드를 컴포넌트 코드와 분리할수도 있구요 😊

```suggestion
import clsx from "clsx";

const Badge = ({ text, color }) => {
  return (
    <span
      className={clsx(styles.badge, styles[color])}
```
```css
/* Badge.module.css */
.primary: {
  background-color: var(--color-blue-100);
  color: var(--color-blue-500);
}
```